CogniteSdk.Types/Alpha/Simulators/SimulatorModelRevisionData.cs

```diff
@@ -0,0 +1,291 @@
+        /// </summary>
+        public string SourceId { get; set; }
+
+        /// <summary>
+        /// Target node identifier.
+        /// </summary>
+        public string TargetId { get; set; }
+
+        /// <summary>
+        /// Type of connection.
+        /// </summary>
+        public SimulatorModelRevisionDataConnectionType ConnectionType { get; set; }
+    }
+
+    /// <summary>
+    /// Thermodynamic data for simulator model revision.
+    /// </summary>
+    public class SimulatorModelRevisionDataThermodynamic
+    {
+        /// <summary>
+        /// IEnumerable of property packages.
+        /// </summary>
+        public IEnumerable<string> PropertyPackages { get; set; }
+
+        /// <summary>
+        /// IEnumerable of components.
+        /// </summary>
+        public IEnumerable<string> Components { get; set; }
+    }
+
+    /// <summary>
+    /// Flowsheet data for simulator model revision.
+    /// </summary>
+    public class SimulatorModelRevisionDataFlowsheet
+    {
+        /// <summary>
+        /// IEnumerable of flowsheet nodes.
+        /// </summary>
+        public IEnumerable<SimulatorModelRevisionDataObjectNode> SimulatorObjectNodes { get; set; }
+
+        /// <summary>
+        /// IEnumerable of flowsheet edges.
+        /// </summary>
+        public IEnumerable<SimulatorModelRevisionDataObjectEdge> SimulatorObjectEdges { get; set; }
+
+        /// <summary>
+        /// Thermodynamic data.
+        /// </summary>
+        public SimulatorModelRevisionDataThermodynamic Thermodynamics { get; set; }
+    }
+
+    /// <summary>
+    /// Base data for simulator model revision.
+    /// </summary>
+    public class SimulatorModelRevisionData
+    {
+        /// <summary>
+        /// External id of the model revision.
+        /// </summary>
+        public string ModelRevisionExternalId { get; set; }
+
+        /// <summary>
+        /// Flowsheet of the model revision.
+        /// </summary>
+        public SimulatorModelRevisionDataFlowsheet Flowsheet { get; set; }
+
+        /// <summary>
+        /// Additional simulator-specific information.
+        /// </summary>
+        public Dictionary<string, string> Info { get; set; }
+
+        /// <summary>
+        /// The number of milliseconds since epoch.
+        /// </summary>
+        public long CreatedTime { get; set; }
+
+        /// <summary>
+        /// The number of milliseconds since epoch.
+        /// </summary>
+        public long LastUpdatedTime { get; set; }
+    }
+
+    /// <summary>
+    /// Retreive class for simulator model revision data.
+    /// </summary>
+    public class SimulatorModelRevisionDataRetrieve
+    {
+        /// <summary>
+        /// External id of the model revision.
+        ///
+        /// </summary>
+        /// <value>External id of the model revision.</value>
+        /// <example>model-revision-external-id</example>
+        /// 
+        public string ModelRevisionExternalId { get; set; }
+
```

Comment:
needs to be on every class, basically
```suggestion
        public string ModelRevisionExternalId { get; set; }
 
        public override string ToString() => Stringable.ToString<SimulatorModelRevisionDataRetrieve>(this);
```